src/tests/ftest/pool/RebuildTests.py

```diff
@@ -236,118 +201,69 @@ def test_multipool_rebuild(self):
 
         :avocado: tags=pool,rebuild,rebuildmulti
         """
-
-        # the rebuild tests need to redo this stuff each time so not in setup
-        # as it usually would be
-        setid = self.params.get("setname", '/run/testparams/setnames/')
-        server_group = self.params.get("server_group",
-                                       '/server/',
-                                       'daos_server')
-
-        basepath = os.path.normpath(self.build_paths['PREFIX']  + "/../")
-        tmp = self.build_paths['PREFIX'] + '/tmp'
-
-        self.hostlist = self.params.get("test_machines", '/run/hosts/')
-        hostfile = WriteHostFile.WriteHostFile(self.hostlist, tmp)
-
         try:
-            ServerUtils.runServer(hostfile, server_group, basepath)
-
-            # use the uid/gid of the user running the test, these should
-            # be perfectly valid
-            createuid = os.geteuid()
-            creategid = os.getegid()
-
-            # parameters used in pool create that are in yaml
-            createmode = self.params.get("mode", '/run/testparams/createmode/')
-            createsetid = self.params.get("setname",
-                                          '/run/testparams/createset/')
-            createsize = self.params.get("size", '/run/testparams/createsize/')
-
             # initialize python pool object then create the underlying
             # daos storage, the way the code is now the pools should be
             # on the same storage and have the same service leader
-            pool1 = DaosPool(self.Context)
-            pool2 = DaosPool(self.Context)
-            pool1.create(createmode, createuid, creategid,
-                         createsize, createsetid, None)
-            pool2.create(createmode, createuid, creategid,
-                         createsize, createsetid, None)
+            pool1 = DaosPool(self.context)
+            pool2 = DaosPool(self.context)
+            pool1.create(self.createmode, self.createuid, self.creategid, self.createsize,
+                         self.createsetid)
+            pool2.create(self.createmode, self.createuid, self.creategid, self.createsize,
```

Comment:
(pylint-line-too-long) Line too long (90/80)

Reply:
no belong to current patch.